modules/sfp_c99.py

```diff
@@ -0,0 +1,624 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_c99
+# Purpose:      SpiderFoot plug-in that queries c99 API
+#
+# Author:      Filip Aleksić <faleksicdev@gmail.com>
+#
+# Created:     2020-08-27
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+from sflib import SpiderFootPlugin, SpiderFootEvent
+
+
+class sfp_c99(SpiderFootPlugin):
+    """C99:Footprint,Passive,Investigate:Content Analysis:apikey:Obtain data from c99 API which offers various data (geo location, proxy detection, phone lookup, etc)."""
+
+    meta = {
+        "name": "C99",
+        "summary": "This module queries c99 API that offers various data (geo location, proxy detection, phone lookup, etc).",
+        "flags": ["apikey"],
+        "useCases": ["Footprint", "Passive", "Investigate"],
+        "categories": ["Content Analysis"],
+        "dataSource": {
+            "website": "https://api.c99.nl/",
+            "model": "COMMERCIAL_ONLY",
+            "references": ["https://api.c99.nl/api_overview", "https://api.c99.nl/faq"],
+            "apiKeyInstructions": [
+                "Visit api.c99.nl",
+                "Press shop in navigation or go to https://api.c99.nl/dashboard/shop",
+                "Press purchase key on option 'C99.NL API KEY' (you can also buy 1 year key if you want)",
+                "You should get your key on your email'",
```

Comment:
"You will receive your API key by email." Also note the hanging single quote at the end.